app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <h2 class="my-4 text-color">Add a New Transaction</h2>
+  <%= form_for @transaction, url: category_transactions_path(@category) do |f| %>
+    <div class="form-group">
+      <%= f.label :name, class: 'text-color' %>
+      <%= f.text_field :name, class: 'form-control', required: true %>
+    </div>
+    
+    <div class="form-group">
+      <%= f.label :amount, class: 'text-color' %>
+      <%= f.text_field :amount, class: 'form-control', required: true %>
+    </div>
+
+    <div class="form-group">
+      <%= f.label :category_ids, "Categories", class: 'text-color' %>
+  <%= f.collection_select :category_ids, Category.all, :id, :name, {  }, { multiple: true, class: 'form-control' } %>

```

Comment:
**OPTIONAL** - Good job rendering the categories for users to select, but you seem to be rendering all categories instead of the categories belonging to a user. This would create an issue when a current_user creates a transaction for a category he did not create since you would not be able to redirect the user to the transaction page for that category. Please fix this 🙏. I suggest only rendering all categories created by the current user.